const throwErr = (status, message) => ({ status, message });

module.exports = { throwErr };

// Função de lançar erros retirada do repositório de aulas da Trybe:
// https://github.com/tryber/sd-013-c-live-lectures/pull/8/commits/b14be5c2060ab40eb30781bcc66d18f594857162
